distro/pom.xml

```diff
@@ -107,6 +107,14 @@
                 <resource>
                   <directory>${project.build.directory}/ozone</directory>
                   <excludes>
+                    <exclude>distro/configs/**/concepts/*.*</exclude>
+                    <exclude>distro/configs/**/drugs/*.*</exclude>
+                    <exclude>distro/configs/**/autogenerationoptions/*.*</exclude>
+                    <exclude>distro/configs/**/globalproperties/*i18n*.*</exclude>
+                    <exclude>distro/configs/**/globalproperties/*drugorder*.*</exclude>
+                    <exclude>distro/configs/**/idgen/*.*</exclude>
+                    <exclude>distro/configs/**/ocl/*.*</exclude>                    
+                    <exclude>distro/configs/**/patientidentifiertypes/*.*</exclude>
```

Comment:
@jnsereko , please do not exclude frontend configurations from Ozone, we need the one for SSO. L121

Reply:
I only exclude the `drugorder` file ozone.